diff -Naur a/matchbox/theme-engines/mb-wm-theme-png.c b/matchbox/theme-engines/mb-wm-theme-png.c
--- a/matchbox/theme-engines/mb-wm-theme-png.c	2018-05-19 02:01:53.375111233 +0100
+++ b/matchbox/theme-engines/mb-wm-theme-png.c	2018-05-19 02:02:52.793309319 +0100
@@ -1133,7 +1133,7 @@
       return NULL;
     }
 
-  if (setjmp (png_ptr->jmpbuf))
+  if (setjmp (png_jmpbuf(png_ptr)))
     {
       png_destroy_read_struct( &png_ptr, &info_ptr, NULL);
       fclose(fd);
