--- a/mimedir/mimedir-attribute.h	2007-03-31 21:16:48.000000000 +0100
+++ b/mimedir/mimedir-attribute.h	2018-03-24 02:26:36.680082211 +0000
@@ -22,6 +22,7 @@
 #define __MIMEDIR_ATTRIBUTE_H__
 
 #include <stdlib.h>
+#include <stddef.h>
 #include <time.h>
 
 #include <glib.h>
@@ -163,7 +164,7 @@
 MIMEDirAttributeType
 		 mimedir_attribute_get_attribute_type		(MIMEDirAttribute *attribute);
 
-ssize_t		 mimedir_attribute_parse			(MIMEDirAttribute *attribute, const gchar *string, GError **error);
+size_t		 mimedir_attribute_parse			(MIMEDirAttribute *attribute, const gchar *string, GError **error);
 gboolean	 mimedir_attribute_write_to_channel		(MIMEDirAttribute *attribute, GIOChannel *channel, GError **error);
 
 #endif /* __MIMEDIR_ATTRIBUTE_H__ */
