--- a/configure.ac	2007-06-06 12:30:47.000000000 +0100
+++ b/configure.ac	2018-05-21 15:00:55.758991032 +0100
@@ -28,18 +28,13 @@
 ##       enable_sicons=$enableval, enable_sicons=no)
 
 
-dnl ---- Check for libmb so we can figure out PNG / XPM support ------------
+dnl ---- Enable PNG support ------------
 
-PKG_CHECK_MODULES(LIBMB, libmb >= 1.1,, AC_MSG_ERROR([*** libmb required and not found ***]))
+AC_ARG_ENABLE(png_icons,
+  [  --enable-png-icons  Use PNG icons ( Rather than XPM icons )],
+     enable_png_icons=$enableval, enable_png_icons=no)
 
-# below is quite bad ...
-if $PKG_CONFIG --libs libmb | grep png ; then
-   found_png="yes"
-else
-   found_png="no"	
-fi
-
-AM_CONDITIONAL(WANT_PNG, test x$found_png = xyes)  
+AM_CONDITIONAL(WANT_PNG, test x$enable_png_icons = xyes)  
 
 ## AM_CONDITIONAL(WANT_SMALL_ICONS, test x$enable_sicons = xyes)  
 
@@ -70,6 +65,6 @@
         prefix:                 ${prefix}
         source code location:   ${srcdir}
 
-	Installing PNGs         ${found_png}
+	Installing PNG Icons    ${enable_png_icons}
 	Installing PDA Folders  ${enable_pda_folders}
 "
