diff -Naur a/configure.ac b/configure.ac
--- a/configure.ac	2018-05-12 13:33:32.018726299 +0100
+++ b/configure.ac	2018-05-19 02:33:34.505324133 +0100
@@ -30,13 +30,13 @@
 
 dnl ---- Check for libmb so we can figure out PNG / XPM support ------------
 
-PKG_CHECK_MODULES(LIBMB, libmb >= 1.1,, AC_MSG_ERROR([*** libmb required and not found ***]))
+PKG_CHECK_MODULES(LIBMATCHBOX,libmatchbox2,, AC_MSG_ERROR([*** libmbatchbox required and not found ***]))
 
 # below is quite bad ...
-if $PKG_CONFIG --libs libmb | grep png ; then
+if $PKG_CONFIG --libs libmatchbox2 | grep png ; then
    found_png="yes"
 else
-   found_png="no"	
+   found_png="no"
 fi
 
 AM_CONDITIONAL(WANT_PNG, test x$found_png = xyes)  
diff -Naur a/matchbox-session b/matchbox-session
--- a/matchbox-session	2018-05-12 13:33:32.019726303 +0100
+++ b/matchbox-session	2018-05-13 00:37:32.986304277 +0100
@@ -4,21 +4,25 @@
 #
 
 # Uncomment below to enable parsing of debian menu entrys
-# export MB_USE_DEB_MENUS=1 
-
-if [ -e $HOME/.matchbox/session ]
-then
-exec $HOME/.matchbox/session
-fi
-
-if [ -e /etc/matchbox/session ]
-then
-exec /etc/matchbox/session
-fi
-
+# export MB_USE_DEB_MENUS=1
+#
+# if [ -e $HOME/.matchbox/session ]
+# then
+# exec $HOME/.matchbox/session
+# fi
+#
+# if [ -e /etc/matchbox/session ]
+# then
+# exec /etc/matchbox/session
+# fi
+#
 # Default files to run if $HOME/.matchbox/session or /etc/matchbox/session
-# dont exist. 
+# dont exist.
+
+START_APPLETS=showdesktop,windowselector
+END_APPLETS=clock,keyboard,systray,startup-notify,notify
 
 matchbox-desktop &
-matchbox-panel  --orientation south &
-exec matchbox-window-manager $@
+matchbox-panel  --edge=bottom --start-applets=$START_APPLETS --end-applets=$END_APPLETS &
+matchbox-keyboard --daemon &
+exec matchbox-window-manager-2-simple -use_cursor yes $@
